konsent/__init__.py

```diff
@@ -551,8 +560,24 @@ def list_unions():
         result.append((union.union_name, union.union_name))
     return result
 
+# list who voted on issue
+def list_who_voted(id, type):
+
+    if type == "post":
+        votes = Vote.query.filter(Vote.post_id == id).all()
+    else:
+        votes = Vote.query.filter(Vote.comment_id == id).all()
+
+    # add votes to tuple
+    result = []
+    for vote in votes:
+        author = User.query.filter(User.id == vote.author_id).one()
+        app.logger.info(author)
```

Comment:
is that debugging leftover?

Reply:
The logging is, yeah. I'll get rid of it, thanks!